packages/@n8n/task-runner/src/task-runner.ts

```diff
@@ -88,6 +88,18 @@ export abstract class TaskRunner {
 			},
 			maxPayload: opts.maxPayloadSize,
 		});
+
+		this.ws.addEventListener('error', (error) => {
+			if (error.message?.includes('ECONNREFUSED') || error.message?.includes('ENOTFOUND')) {
```

Comment:
We could check `error.code` instead of using `message`

Reply:
[85e9c72](https://github.com/n8n-io/n8n/pull/11635/commits/85e9c72acab3d183eca90385db36222e9df78172)